index.html

```diff
@@ -2128,6 +2228,216 @@ <h4>Bidirectional text</h4>
 
     </section> <!-- Profile Constraints -->
 
+    <section id="ttml-to-data-model">
+      <h2>Mapping from TTML to the DAPT Data Model</h2>
+      <p class="ednote">Consider renaming to "Special provisions for processors that read, modify and write DAPT documents"
+        or something pithier.</p>
+      <p>It is possible that conformant <a>DAPT documents</a> contain entities that
+        do not map unambiguously, or at all, to the model defined at <a href="#data-model"></a>.
+        This could be because it has been generated by a processor conformant to some future
+        version of DAPT, or through a generic [[TTML2]] process.</p>
+      <p>This section defines the processing behaviour for processors that read conformant <a>DAPT documents</a>,
+        so that they can interoperably handle unrecognised entities.
+        The intent is to pave the way forwards so that processors conformant to earlier versions of DAPT
+        can process documents conformant to later versions,
+        while retaining the semantics of the earlier version.</p>
+      <div class="note">
+        <p>These provisions apply especially to implementers of DAPT processers that
+        populate a data structure corresponding
+        to the DAPT data model from a conformant input <a>DAPT document</a>.
+        There is no generalised requirement for DAPT processors to map DAPT documents to the DAPT data model,
+        as long as their output behaviour is consistent with the provisions of this specification.</p>
+        <p>For example, a generic [[TTML2]] processor can process a TTML <a>document instance</a>
+        corresponding to the representation defined in <a href="#data-model"></a> without
+        performing such a mapping, especially for presentation purposes such as
+        rendering an audio mix based on the contents of the document.</p>
```

Comment:
I'm happy to try to simplify the phrasing, but I'm worried that throwing in "2 categories of processor" will make things worse rather than better - I worry that folk are going to want to see definitions for these categories, and I don't think the spec complexity is worth the small possible benefit. The future compatibility bit is already discussed in the next paragraph, so I wouldn't want to duplicate it.

By the way, would it help if instead of "unrecognised entities" we use one of:
* "unrecognised XML entities" or
* "unrecognised vocabulary" or
* "unrecognised attributes or elements"?

Reply:
I'm going to go with the most explicit of those three, "unrecognised attributes or elements" for now, happy to revisit if there's a better option.

Reply:
Massively simplified this whole note in f8c86e4 to say:

```html
It is also possible to process <a>DAPT documents</a> using generic [[TTML2]] processors,
which do not necessarily map the documents to the DAPT data model.
For example a generic TTML2 processor could render an audio mix based on
a DAPT document without needing to model <a>Script Events</a> <i>per se</i>.
```